packages/E2ETest/run_wdio.js

```diff
@@ -54,22 +54,6 @@ function selectSpecs(folder) {
 let opts = selectSpecs(specFolder);
 console.log(`Selected tests: ${opts}`);
 
-function ensureRunningInHyperV() {
```

Comment:
Removed since tests run fine outside of a VM, so we see a prompt before running tests that isn't super useful.